README.md

```diff
@@ -29,21 +29,38 @@ so you can easily execute the command line utilities that are described later in
 
 ## Installing
 
-Create and activate a virtual environment within which you can install the package:
+Create and activate a Python 3.9+ virtual environment within which you can install the package:
 
 ```shell
-python -m venv .venv
+python3 -m venv .venv
 source .venv/bin/activate
+pip install poetry
+pip install python-dotenv
```

Comment:
Hm, I think these might be unnecessary?

In the [Installation Requirements and Pre requisites](https://github.com/cancerDHC/sheet2linkml/tree/issue-1-readme#installation-requirements-and-pre-requisites) section of the README there's already a link to `poetry` which has instructions on how you can install `poetry` on your machine through the executable.

Also, we don't need the second line. The correct way to install the dependencies from the repository is by running:

```
poetry install
```

And this will use the dependencies from the lock file to initialize the virtual environment created in the steps immediately above.

Reply:
I could not get your code to run without these two additional `pip install`s

Reply:
You can follow the poetry installation instructions [here](https://python-poetry.org/docs/#installation), specifically the _osx / linux / bashonwindows install instructions_ section.

Once that's done, you can create a virtual environment, and run:

```
poetry install
```

within it.

Reply:
OK, that's all clear now. Thanks.